doc/refman/Classes.tex

```diff
@@ -400,18 +444,105 @@ \subsection{\tt Typeclasses Transparent, Opaque {\ident$_1$ \ldots \ident$_n$}}
 
 This is equivalent to {\tt Hint Transparent,Opaque} {\ident} {\tt: typeclass\_instances}.
 
+\subsection{\tt Set Typeclasses Module Eta}
+\optindex{Typeclasses Modulo Eta}
+
+This allows eta-conversion during unification of type-classes.
```

Comment:
It is both actually, detailing. Also, this is not working with Filtered Unification on (it always does eta then), I detailed the doc.